src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -230,8 +245,12 @@ object RuntimeAttributes {
     astList.asScala.toVector map { a => processRuntimeAttribute(a.asInstanceOf[Ast]) } toMap
   }
 
-  private case class LocalDisk(name: String, sizeGb: Long, diskType: String) {
-    def toDisk: Disk = new Disk().setSizeGb(sizeGb).setType(diskType).setName(name)
+  private case class LocalDisk(name: String, diskType: DiskType, sizeGbOption: Option[Long] = None) {
+    def toDisk: Disk = {
+      val disk = new Disk().setName(name).setType(diskType.googleTypeName).setAutoDelete(true)
+      sizeGbOption.foreach(sizeGb => disk.setSizeGb(sizeGb)) // scala.Long to java.lang.Long
```

Comment:
This one doesn't work unless I do more conversion elsewhere. sizeGb is a `scala.Long`. `setSizeGb` takes a `java.lang.Long`.